documentation/rpi3.md

```diff
@@ -216,8 +208,6 @@ We're just going to create symlinks. By doing so you don't have to think about
 copy files, just rebuild and you have the latest version available for the next
 boot. On my computer I've symlinked like this (in my `/tftpboot` folder):
 ```
-$ ll
-lrwxrwxrwx  1 jbech  jbech         65 jul 14 09:03 Image -> /home/jbech/devel/optee_projects/rpi3/linux/arch/arm64/boot/Image
```

Comment:
I've added it again, I still have no idea how that disappeared (probably a bit too much `vim + dd').